packages/fluentui/react-northstar/src/components/Pill/Pill.tsx

```diff
@@ -76,6 +78,10 @@ export const Pill: ComponentWithAs<'span', PillProps> & FluentComponentStaticPro
   const { setStart, setEnd } = useTelemetry(Pill.displayName, context.telemetry);
   setStart();
 
+  const parentProps = (useContextSelectors(PillContext, {
+    role: v => v.role,
+  }) as unknown) as PillSubscribedValue; // TODO: we should improve typings for the useContextSelectors
```

Comment:
is context selector necessary over native context at this stage ? It seems that with memoization, the values in this context are not going to change frequently.

If there is no other value going to be in this context you could even use the native context with a string (primitive) value then even memoization wouldn't bee needed.

Reply:
Not really, native context should do the work. BUT I can't recall any of our components using native context so I used the context selectos for consistency?!